lisa/sut_orchestrator/libvirt/schema.py

```diff
@@ -47,16 +48,48 @@ class QemuPlatformSchema:
     network_boot_timeout: Optional[float] = None
 
 
+# Possible disk image formats
+class DiskImageFormat(Enum):
+    QCOW2 = "qcow2"
+    RAW = "raw"
+
+
 # QEMU orchestrator's per-node configuration options.
 @dataclass_json()
 @dataclass
-class QemuNodeSchema:
-    # The disk image to use for the node.
-    # The file must use the qcow2 file format and should not be changed during test
-    # execution.
-    qcow2: str = ""
+class BaseLibvirtNodeSchema:
+    # Path to the OS disk image to use for the node.
+    disk_img: str = ""
+    # Format of the disk image specified above.
+    disk_img_format: str = ""
     # Configuration options for cloud-init.
     cloud_init: Optional[CloudInitSchema] = None
     # Whether to use UEFI or BIOS firmware.
     # Defaults to UEFI.
     firmware_type: str = ""
+
+
+# QEMU orchestrator's per-node configuration options.
+# This ensures backward compatibility with existing runbooks that specify the
+# qcow2 property.
+@dataclass_json()
+@dataclass
+class QemuNodeSchema(BaseLibvirtNodeSchema):
+    # The disk image to use for the node.
+    # The file must use the qcow2 file format and should not be changed during test
+    # execution.
+    qcow2: str = ""
```

Comment:
`qcow2` field is no more required. `disk_img` field of `BaseLibvirtNodeSchema` class already handles this.

Reply:
This is for backward compatibility with existing qemu runbooks.